src/lib.rs

```diff
@@ -20,16 +20,14 @@
 extern crate freetype_sys;
 extern crate libc;
 
-/// A wrapper over FT_Error so we can add convenience methods on it.
-#[repr(C)]
-#[derive(Copy, Clone, PartialEq, Eq, Debug)]
-pub struct FT_Error(pub ::std::os::raw::c_int);
+pub type FT_Error = ::std::os::raw::c_int;
 
-impl FT_Error {
-    #[inline]
-    pub fn succeeded(&self) -> bool {
-        self.0 == freetype::FT_Err_Ok as ::std::os::raw::c_int
-    }
+pub trait FTErrorMethods {
```

Comment:
Please remove that trait if we don't want `FT_Error` to be a wrapper.

Reply:
I am going to make to free function.